### ISSUE TITLE
[FR] Suggestion to Add a Note in the “Prep” Window Regarding Progress Bar Behavior

### ISSUE THREAD

**Issue by adonboli** [grahampugh/erase-install#522]
Hello,

I believe it would be helpful to include a note in the “prep” window explaining that the progress bar may appear to stall for a period (typically around 54% or 56%), and that this is normal and the update is not stuck but happening in the background. 

My understanding is that this is a normal occurrence, as the startosinstall process often pauses progress reporting before suddenly jumping to 100%, or even occasionally over 100%.

This isn’t an issue with the erase-install process, which is why I’m not filing it as a bug. However, we’ve noticed that this behavior can cause users to think that the update has failed or become stuck, when in reality, the process is still running, just without updating the progress percentage.

If you need screenshots to clarify this, I can provide them to help make this more understandable.

**Comment by grahampugh** [comment]
Yes I've seen this in the past few months, not ideal but it's purely reading the output from startosinstall. I agree it might need a note. It might also be worth lodging a FB with Apple.

**Comment by adonboli** [comment]
I will file a feedback with Apple, hopefully they will fix it.

**Comment by grahampugh** [comment]
I will check this now that Sequoia is out. The problem with adding this message is that it needs to be translated into 8 languages...
